docs/faq/multi_devices.md

```diff
@@ -82,124 +82,7 @@ Note that this option may result in higher GPU memory usage.
 
 When using a large number of GPUs, e.g. >=4, we suggest using `device` for better performance.
 
-## Distributed Training with Multiple Machines
+## Multiple devices across machines
```

Comment:
?

Reply:
Just for uniformity with the above section which says 'Multiple GPUs within a single machine'. I can change it back if you that that is better

Reply:
So you have several devices attached to several machines?

How about Distributed Multi-node GPU Training? Implies many machines with a single GPU...
Or Distributed Multi-node Multi-GPU Training? Implies many machines with multiple GPUs per machine.

I believe we set this heading this way because we could also deal with CPU training here...

In absence of doubt, clarify by adding a definition immediately after this...